modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -487,7 +487,16 @@ private CompletableFuture<Void> completeRebalance(@Nullable Throwable throwable)
                     raftGroupConfig
             );
 
-            return partitionSnapshotStorage.partition().finishRebalance(meta.lastIncludedIndex(), meta.lastIncludedTerm(), raftGroupConfig);
+            return partitionSnapshotStorage.partition().finishRebalance(
+                    new AccessPartitionMeta(
```

Comment:
I think we can add a constructor/static method to create a new `AccessPartitionMeta` based on `PartitionSnapshotMeta`.